### DISCUSSION TITLE
de/1-08-ai-and-society/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#59] [General]
# de/1-08-ai-and-society/

https://ai.lernos.org/de/1-08-ai-and-society/

**Comment by eejit0703** [comment]
Schreibfehler unter den 'Aspekten': Naschvollziehbarkeit  ;-)

**Reply by simondueckert** [reply]
Ein sehr schöner Typo 😂

**Comment by eejit0703** [comment]
Schreibfehler in der Überschrift: (Verzerrungen und) Diskrimierungen

**Comment by eejit0703** [comment]
Schreibfehler in 'Ethik und Wertesystem': (...oder unerwünschte Antworten aus Chatbots) heraufiltern

**Comment by eejit0703** [comment]
Schreibfehler in 'Regulierung': (...Interessenausgleich zwischen dem Ausschöpfen der Innovationspotenziale und der) Risikomimierung

**Comment by inamakie** [comment]
zwei formale Details:  die Abkürzung für „zum Beispiel“ lautet „z. B. “. Wichtig ist, dass zwischen den Buchstaben ein Leerzeichen steht.
Ich fände hilfreich und orientierend, wenn die Reflexionsfragen nummeriert wären, dann fällt es beim Bearbeiten und dem Austausch in der Gruppe leichter sich darauf zu beziehen.

**Comment by MariaLizMatthaeus** [comment]
Als kleine Gedankenstütze für das Kapitel "KI & Gesellschaft" ist es wertstiftend 

- KI & Umwelt + Implikationen auf die Nachhaltigkeit/Ressourcenverbrauch

Raum zu geben. 

Dieser Gedanke wurde von verschiedenen Stimmen geäußert und mehrfach diskutiert, u.a. auf LinkedIn und in den Learning Circles. 
Ich mag ihn hier direkt im Leitfaden festhalten und niederschreiben.

**Comment by SylwiaMiedza** [comment]
"Neue Tätigkeiten und Berufe werden entstehen, im Datenmanagement, im KI-Trainingsbereich". 
==> hier das letzte Komma sollte mit "oder" ersetzt werden.

"Insgesamt könnte es jedoch eine neue Polarisierung zwischen den qualifizierten Kräften, die die KI für sich nutzen und einsetzen geben und denen, die durch Nichtnutzung ihren "Marktwert" verlieren." 
==> Zwischen "einsetzen" und "geben" muss ein Komma eingesetzt werden.

**Comment by SylwiaMiedza** [comment]
Das gesamte Dokument sollte daraufhin geprüft werden, ob die Pronomen "Du", "Dir", "Dich", "Dein", etc. entweder groß- oder kleingeschrieben werden. Derzeit findet man beide Schreibweisen im Dokument. Dies sollte möglichst einheitlich ein.

**Comment by SylwiaMiedza** [comment]
Zum Thema Verzerrungen...

In diesem Kapitel wurde weitgehend auf Geschlechter-neutrale Formulierungen geachtet. Dies ist in dem Leitfaden nicht durchgehend eingehalten worden. Es sollte möglichst einheitlich (am besten Gender-neutral) formuliert sein. Unter diesem Aspekt sollte der Leitfaden überprüft werden. 

Beispiele:

Gender-neutrale Formulierung:
"Es ist daran zu denken, dass viele Anbietende insbesondere kostenloser KI-Tools die Eingaben der Nutzenden für das Training ihrer Modelle verwenden." 

Gender-spezifische Formulierung:
"Häufig diskutiert wird der Einsatz von KI-Systemen bei der Bewerberauswahl ..."
"Bevorzuge Anbieter von KI-Plattformen..."

**Comment by KathaZ** [comment]
A2/Z7: einsetzen*,* geben
Generell: Anführungszeichen oben in einheitlicher Typo in allen Kapiteln
Generell: Gedankenstriche bei Einschüben als Halbgeviert setzen (–)
A4/Z2: *Auswahl von Bewerber:innen*
A5/Z4: unbewu*ss*t 
A6/Z8: Problemlösung *etc.*
A7/Z2: sollte: *d*em
A7/Z5: *R*und-um-die-Uhr-Geschäfts
A7/Z7,8: und wer *dafür* rechenschaftspflichtig *ist*,
Wechsel bei den Reflexionsfragen zwischen z. B. "deinem" und "meinem" Unternehmen: Sollte einheitlich sein, um Verwirrung zu vermeiden

**Comment by ikomarek** [comment]
Müsste hier nicht auch zwingend der ökologische Aspekt hinein? KI Nutzung ist ja im Vergleich zu anderen digitalen Anwendungen, wie z.B. googlen, sehr ressourcenintensiv (um es milde auszudrücken).
